packages/cloud-cognitive/src/components/CreateTearsheet/CreateTearsheet.js

```diff
@@ -97,7 +98,11 @@ export let CreateTearsheet = forwardRef(
       if (lastItem !== lastIncludedStep) {
         setLastIncludedStep(lastItem);
       }
-    }, [stepData, firstIncludedStep, lastIncludedStep]);
+      if (open && initialStep) {
+        let numberOfHiddenSteps = getNumberOfHiddenSteps(stepData, initialStep);
```

Comment:
Change this `let` to `const` since it doesn't need to be reassigned
```suggestion
        const numberOfHiddenSteps = getNumberOfHiddenSteps(stepData, initialStep);
```